matterpy/contrib/rss.py

```diff
@@ -72,10 +78,18 @@ def init(manager, conf):
 
 
 def post_to_text(entry, format_string):
-    title = entry.title
-    url   = entry.link
-    body  = entry.description
+    try:
+        return format_string.format(
+            url = entry.link,
+            body = entry.summary,
```

Comment:
`summary`? Don't you mean `description`?

Reply:
There is actually no ```description``` element for the entrys, only for the Channel itself.
Or it is just not as common, ```summary``` is more common.